sdks/python/setup.py

```diff
@@ -314,9 +314,7 @@ def get_portability_package_data():
             'google-cloud-bigquery>=2.0.0,<4',
             'google-cloud-bigquery-storage>=2.6.3,<3',
             'google-cloud-core>=2.0.0,<3',
-            # TODO(https://github.com/apache/beam/issues/26673)
-            # 2.18.x breaks unit test
-            'google-cloud-bigtable>=2.0.0,<2.18.0',
+            'google-cloud-bigtable>=2.19.0,<3',
```

Comment:
@tvalentyn is it safe to change the lower bound for this package?

Reply:
FYI, the updates in bigtableio.py will break with older version of google-cloud-bigtable :(. And seems like the lower boundary was updated for some other modules before (e.g. bigquery-storage)

Reply:
Yes, increasing the lower bound is appropriate and necessary in this case.